### PR TITLE
Update the class definition BBE

### PR DESCRIPTION
**Pull request by MaryamZi**
## Purpose
$title.

Updated to include information about the `init` method and other methods.

### CODE REVIEW

**Review by rdhananjaya — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

File: examples/class-definition/class_definition.bal [hunk 1]
```diff
@@ -1,30 +1,126 @@
...
+// method, if present.
 class Person {
+    public string name;
+    public int age;
+    public string email = "default@abc.com";
+
+    // Fields with no access modifiers are visible only within the same module.
+    string address = "No 20, Palm Grove";
+
+    // Private fields are only visible within the object.
```
Comment (pubudu91) [inline examples/class-definition/class_definition.bal:14]: object or class? :thinking:

Comment (MaryamZi) [reply examples/class-definition/class_definition.bal:14]: Yeah, it should be class here. Thanks!

Comment (pubudu91) [reply examples/class-definition/class_definition.bal:14]:
```suggestion
    // Private fields are only visible within the class.
```

File: examples/class-definition/class_definition.bal [hunk 2]
```diff
@@ -1,30 +1,126 @@
...
+        self.name = name;
+        self.age = age;
+        self.employed = employed;
+    }
+
+    // Methods can access all the fields of the class.
+    function isEmployed() returns boolean => self.employed;
+}
+
+// Define a class named `Person`. It contains `public`, `private`, and module-level visible fields.
```
Comment (pubudu91) [inline examples/class-definition/class_definition.bal:36]:
```suggestion
// Define a class named `Employee`. It contains `public`, `private`, and module-level visible fields.
```

File: examples/class-definition/class_definition.bal [hunk 3]
```diff
@@ -1,30 +1,126 @@
...
-    // There are three ways to initialize a `Person` object.
-    Person p1 = new;
-    io:println(p1.age);
+    // If there is no `init` method or the `init` method does not accept parameters,
+    // a value can be constructed with just the `new` expression.
+    Employee e1 = new;
+    io:println(e1.age);
+
+    // Such a value can also be constructed with the `new` expression with empty parentheses.
+    Employee e2 = new ();
```
Comment (pubudu91) [inline examples/class-definition/class_definition.bal:80]: Don't we usually format this type of new expression as `new()`?

Comment (MaryamZi) [reply examples/class-definition/class_definition.bal:80]: No, I think we decided to leave a space for consistency with `new Employee()`.

**Review by pubudu91 — APPROVED** [review]

**Pull request merged**